client/gutenberg/extensions/tiled-gallery/index.js

```diff
@@ -118,6 +118,36 @@ export const settings = {
 	title: __( 'Tiled Gallery' ),
 	transforms: {
 		from: [
+			{
+				type: 'block',
+				isMultiBlock: true,
+				blocks: [ 'core/image' ],
+				transform: attributes => {
+					// Leave out other than supported aligns: image block supports left and right, Tiled Gallery doesn't.
+					const objectsWithValidAlign = filter( attributes, object =>
```

Comment:
What do you think of dropping all of this and just using the default alignment?

It seems like a lot of special handing for very limited benefit for the user. When they apply this transform, it's a pretty radical change in layout. It wouldn't seem surprising that you'd need to make some adjustments if desired.

Reply:
I like that tradeoff. Good idea!

Reply:
https://github.com/Automattic/wp-calypso/pull/30301/commits/9a8966014754b645ec961347c405adebf41aebcf